deja-dup: 38.1 -> 38.3

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/deja-dup/versions.



meta.description for deja-dup is: '"A simple backup tool"'.


<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS
- /nix/store/591akdxgjinsr37nlz2wc6zfkc8wjj8s-deja-dup-38.2/bin/deja-dup passed the binary check.
- /nix/store/591akdxgjinsr37nlz2wc6zfkc8wjj8s-deja-dup-38.2/bin/.deja-dup-wrapped passed the binary check.
- 2 of 2 passed binary check by having a zero exit code.
- 2 of 2 passed binary check by having the new version present in output.
- found 38.2 with grep in /nix/store/591akdxgjinsr37nlz2wc6zfkc8wjj8s-deja-dup-38.2
- directory tree listing: https://gist.github.com/2182f22a62874e4adb66f691e6e80271
- du listing: https://gist.github.com/afaa67315a06bae34088113914c65c29

</details>
<details>
<summary>
Rebuild report (if merged into master) (click to expand)
</summary>

3 total rebuild path(s)

1 package rebuild(s)

1 x86_64-linux rebuild(s)
1 i686-linux rebuild(s)
0 x86_64-darwin rebuild(s)
1 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
deja-dup

</details>

<details>
<summary>
Instructions to test this update (click to expand)
</summary>

Either download from Cachix:
```
nix-store -r /nix/store/591akdxgjinsr37nlz2wc6zfkc8wjj8s-deja-dup-38.2 \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)

Or, build yourself:
```
nix-build -A deja-dup https://github.com/r-ryantm/nixpkgs/archive/cc12c6cfe4c1fb76019d6f8ce1c435dcd1df8d56.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/591akdxgjinsr37nlz2wc6zfkc8wjj8s-deja-dup-38.2
ls -la /nix/store/591akdxgjinsr37nlz2wc6zfkc8wjj8s-deja-dup-38.2/bin
```


</details>
<br/>


cc @jtojnar @joncojonathan for testing.

jtojnar commented:
Weird, why does not it update to 38.3?

worldofpeace commented:
> Weird, why does not it update to 38.3?

oh, missed that. Will bump it again.

Is weird though, sometimes it appears that the update prs are out of date.
Or maybe I'm missing a detail.

worldofpeace committed:
deja-dup: 38.1 -> 38.3, cleanup

This includes:

- fetch source from gitlab
- pname-version stuff
- change homepage
- correct license
  See: https://gitlab.gnome.org/World/deja-dup/blob/master/meson.build#L21

ryantm commented:
[38.3 was released today](https://launchpad.net/deja-dup/+announcement/15198) and I started the updater with a dump from Repology on yesterday.
